MonoGame.Framework/Graphics/SpriteBatch.cs

```diff
@@ -674,52 +674,47 @@ public void DrawString (SpriteFont spriteFont, string text, Vector2 position, Co
                     firstGlyphOfLine = true;
                     continue;
                 }
-
-                if (!spriteFont.Glyphs.TryGetValue(c, out currentGlyph))
-                {
-                    if (!defaultGlyph.HasValue)
-                        throw new ArgumentException(SpriteFont.Errors.TextContainsUnresolvableCharacters, "text");
-
-                    currentGlyph = defaultGlyph.Value;
-                }
+ 
+                var currentGlyphIndex = spriteFont.GetGlyphIndexOrDefault(c, defaultGlyphIndex);
+                var pCurrentGlyph = pGlyphs + currentGlyphIndex;
```

Comment:
Are we sure here that `currentGlyphIndex` can never let us index outside the `SpriteFont.Glyphs` array?